src/frontends/tensorflow/tests/convert_saved_model.cpp

```diff
@@ -114,3 +114,13 @@ TEST_F(TransformationTestsF, SavedModelMultipleTensorNames) {
         model_ref = make_shared<Model>(OutputVector{result}, ParameterVector{x});
     }
 }
+
+TEST_F(TransformationTestsF, SavedModelBroadcastIssue) {
```

Comment:
please use `FrontEndConversionWithReferenceTestsF` and update sources. we no longer use ngraph utils